catalyst/callbacks/quantization.py

```diff
@@ -1,3 +1,54 @@
+from typing import Dict, Optional, TYPE_CHECKING, Union
+from pathlib import Path
+
+import torch
+
+from catalyst.core import Callback, CallbackNode, CallbackOrder
+from catalyst.utils import BestModel, quantize_model
+
+if TYPE_CHECKING:
+    from catalyst.core import IRunner
+
+
+class QuantizationCallback(Callback):
+    def __init__(
+        self,
+        metric: str,
+        minimize_metric: bool = True,
```

Comment:
I am not sure, this callback should know about model selection. We already have `CheckpointCallback` for such logic. The Utils callback could just execute the pruninng/tracing/quantization with `runner.model`